lemur/dns_providers/schemas.py

```diff
@@ -0,0 +1,27 @@
+from lemur.common.fields import ArrowDateTime
+from lemur.common.schema import LemurInputSchema, LemurOutputSchema
+
+from marshmallow import fields
+
+
+class DnsProvidersNestedOutputSchema(LemurOutputSchema):
+    __envelope__ = False
+    id = fields.Integer()
+    name = fields.String()
+    provider_type = fields.String()
+    description = fields.String()
+    credentials = fields.String()
+    api_endpoint = fields.String()
+    date_created = ArrowDateTime()
+
+
+class DnsProvidersNestedInputSchema(LemurInputSchema):
```

Comment:
Are any of the fields required/not-required? Should there be additional validation?